src/ui/party-ui-handler.ts

```diff
@@ -1823,19 +1828,34 @@ class PartySlot extends Phaser.GameObjects.Container {
     partyUiMode: PartyUiMode,
     tmMoveId: MoveId,
   ) {
-    super(
-      globalScene,
-      slotIndex >= globalScene.currentBattle.getBattlerCount() ? 230.5 : 64,
-      slotIndex >= globalScene.currentBattle.getBattlerCount()
-        ? -184 +
-            (globalScene.currentBattle.double ? -40 : 0) +
-            (28 + (globalScene.currentBattle.double ? 8 : 0)) * slotIndex
-        : partyUiMode === PartyUiMode.MODIFIER_TRANSFER
-          ? -124 + (globalScene.currentBattle.double ? -20 : 0) + slotIndex * 55
-          : -124 + (globalScene.currentBattle.double ? -8 : 0) + slotIndex * 64,
-    );
+    const isBenched = slotIndex >= globalScene.currentBattle.getBattlerCount();
+    const isDoubleBattle = globalScene.currentBattle.double;
+    const isItemManageMode = partyUiMode === PartyUiMode.MODIFIER_TRANSFER || partyUiMode === PartyUiMode.DISCARD;
+
+    /**
+    Here we determine the position of the slot.
+    The x coordinate depends on whether the pokemon is on the field or in the bench.
+    The y coordinate is more complex to determine
```

Comment:
```suggestion
    /*
     * Here we determine the position of the slot.
     * The x coordinate depends on whether the pokemon is on the field or in the bench.
     * The y coordinate depends on various factors, such as
```
(And then fill out the end of the comment)